src/components/form/examples/dynamic-form.tsx

```
@@ -45,7 +45,14 @@ export class DynamicFormExample {
 
     public render() {
         return [
-            <textarea onChange={this.handleTextChange}>{this.text}</textarea>,
+            <limel-code-editor
+                language="json"
+                lineNumbers={true}
+                lint={true}
+                fold={true}
+                onChange={this.handleTextChange}
+                value={this.text}
+            />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)